pecan/compat/__init__.py

```diff
@@ -18,3 +18,39 @@
 
 def is_bound_method(ob):
     return inspect.ismethod(ob) and six.get_method_self(ob) is not None
+
+
+def getargspec(func):
+    import sys
+    if sys.version_info < (3, 5):
+        return inspect.getargspec(func)
+
+    sig = inspect._signature_from_callable(func, follow_wrapper_chains=False,
```

Comment:
This private function call is gross, but if I can get http://bugs.python.org/issue27172 merged, we can restrict its usage to py35-only and rely on inspect.Signature.from_callable() in future releases.